ciphers/monoalphabetic_cipher.cpp

```diff
@@ -0,0 +1,50 @@
+#include <bits/stdc++.h>
+using namespace std;
+unordered_map<char,char> hashMap;
+
+string encrypt(string msg)
+{
+  string ciphertext;
+  for(int i=0; i<msg.size(); i++)
+  {
+    ciphertext.push_back(hashMap[msg[i]]);
+  }
+  
+  return ciphertext;
+}
+
+string decrypt(string msg)
+{
+  string plaintext;
+  for(int i=0; i<msg.size(); i++)
+  {
+    plaintext.push_back(hashMap[msg[i]]);
+  }
+  
+  return plaintext;
+}
+
+void hashFn(string a, string b)
+{
+  hashMap.clear();
+  for(int i=0; i<a.size(); i++)
+  {
+    hashMap.insert(make_pair(a[i],b[i]));
+  }
+}
+
```

Comment:
Please add self-test cases so we can verify your program works properly (**see the [typical structure of a program](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program) as a reference**). Let us know if you need any help here or on our [**Discord server**](https://the-algorithms.com/discord). 😃